testapp/src/main/res/layout/settings_menu_activity.xml

```diff
@@ -72,6 +65,14 @@
         android:inputType="text" />
     </com.google.android.material.textfield.TextInputLayout>
 
+    <com.google.android.material.switchmaterial.SwitchMaterial
+        android:id="@+id/switchTestMode"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/small_12"
+        android:text="@string/lb_test_mode"
+        android:visibility="invisible"/>
```

Comment:
Does it need to be invisible? I don't know the spec though.

Reply:
I think this is temporary because Testing API endpoints are not ready yet. We can probably put a TODO here to change it back to visible once they are available.